[feat] add horizontal moving that is acceleration on character

rjwu95 opened the pull request:
- Using speedX and speedY value

rjwu95 committed:
[feat] add horizontal moving that is acceleration on character

 - Using speedX and speedY value

DoheeKang commented on src/app/Game.js:
```diff
@@ -1,6 +1,34 @@
+import Player from "./Player";
+import Control from "./Control";
+const PLAYER_WIDTH = 30
+const PLAYER_HEIGHT = 30
+const PLAYER_COLOR = "red"
```
[의견] 이러한 것들은 constant 파일을 하나 만들어서 분리하면 좋을 것 같아요!

rjwu95 replied:
constant.js 만들었습니다

DoheeKang commented on src/app/Control.js:
```diff
@@ -1,4 +1,37 @@
-class Control {
+export class Control {
+  constructor(game, player) {
+    this.game = game
+    this.player = player
+    this.keys_ = new Map();
```
[질문] game과 keys_가 Control에 있는 이유가 혹시 무엇일까요?

rjwu95 replied:
처음에 사용했었는데 불필요해서 아예 뺐습니다!

DoheeKang commented on src/app/Game.js:
```diff
@@ -1,6 +1,34 @@
+  clear() {
+    this.context.clearRect(0, 0, this.canvas.width, this.canvas.height);
+  }
+  renderPlayer() {
+    const ctx = this.context;
+    ctx.save()
+    ctx.fillStyle = PLAYER_COLOR;
+    ctx.fillRect(this.player.x, this.player.y, PLAYER_WIDTH, PLAYER_HEIGHT);
+    ctx.restore()
+  }
```
[의견] 그냥 의견이긴 한데 해당 renderPlayer는 Player 클래스에 있는게 어떨까요?

rjwu95 replied:
좋습니다 Player로 옮겼습니다

rjwu95 committed:
[refactor] Apply code reviews

 - Create constant.js file
 - Moving renderPlayer method to Player.js
 - Remove variables in Control

rjwu95 committed:
[feat] Add max speed in horizontal moving

 - Temporary max speed value is 3.5 in constant

DoheeKang reviewed (approved):
okay! merge해주세요!

DoheeKang commented:
우선 제가 머지 진행했습니다~
